typescript.js

```diff
@@ -361,6 +361,9 @@ module.exports = {
         'quotes': 'off', // covered by @typescript-eslint/quotes
         'semi': 'off', // covered by @typescript-eslint/semi
         'space-before-function-paren': 'off', // covered by @typescript-eslint/space-before-function-paren
+        'no-undef': 'off', // off because variables in d.ts files will be shown as error otherwise
```

Comment:
TypeScript handles this by it's own see [this](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md#i-get-errors-from-the-no-undef-rule-about-global-variables-not-being-defined-even-though-there-are-no-typescript-errors) 😄

Reply:
This page redirects me to an 404 😅

Reply:
The changed there structure have a look [here](https://github.com/typescript-eslint/typescript-eslint/blob/main/docs/linting/TROUBLESHOOTING.md#i-get-errors-from-the-no-undef-rule-about-global-variables-not-being-defined-even-though-there-are-no-typescript-errors), @JenniferFalkenstein.

Reply:
Just rewrite the comment that typescript handles that on its own.